backend/clients/twilio_client.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+import logging
+import time
+import phonenumbers
+
+from datetime import datetime
+from json.decoder import JSONDecodeError
+from twilio.base.exceptions import TwilioRestException
+from twilio.rest import Client
+from pydantic import BaseModel, ValidationError
+from pydantic_extra_types.phone_numbers import PhoneNumber
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+MAX_WHATSAPP_MESSAGE_LENGTH = 1500
+
+
+class IncomingMessage(BaseModel):
+    MessageSid: str
+    From: str
+    Body: str
+    NumMedia: int
+
+
+class PhoneValidationModel(BaseModel):
+    phone: PhoneNumber
+
+
+class TwilioClient:
+    def __init__(self):
+        self.TWILIO_ACCOUNT_SID = os.getenv("TWILIO_ACCOUNT_SID")
+        self.TWILIO_AUTH_TOKEN = os.getenv("TWILIO_AUTH_TOKEN")
+        self.TWILIO_WHATSAPP_NUMBER = os.getenv("TWILIO_WHATSAPP_NUMBER")
+        if os.getenv("TESTING"):
```

Comment:
Please remove this. This is a task for `.env`.

Reply:
Done